app/controllers/game_controller.rb

```diff
@@ -12,7 +12,9 @@ def init
 
   def turn
     @board = Board.new session['board']
-    @players = PlayerList.new session['players'][0]['name'], session['players'][1]['name'], session['current_player']
+    @players = PlayerList.new session['players'][0]['name'],
+                              session['players'][1]['name'],
+                              session['current_player']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.